### ISSUE TITLE
[feature]: add link to test-reports, show beautified summary report on cli after test results 

### ISSUE THREAD

**Issue by nehagup** [keploy/keploy#785]
### Is there an existing feature request for this?

- [X] I have searched the existing issues

### Summary

Following information should be added:

1. Link to the test-report generated for that particular test-run
2. Test-Summary like Keploy v1

### Why should this be worked on?

To enhance the understanding of the user around the test-cases executed. 

### Repository

keploy

**Comment by nehagup** [comment]
![test-runs-success](https://github.com/keploy/keploy/assets/15074229/6a954219-ca6f-4312-bbdb-ef7e4abdaaa9)

**Comment by ganeshrevadi** [comment]
@nehagup  Can you assign this to me ?
Thanks!

**Comment by nehagup** [comment]
@ganeshrevadi sure!

**Comment by AkashKumar7902** [comment]
Hey @PranshuSrivastava, In the PR i had mentioned that the console doesn't support orange as only 8 colours are possible. But after wandering a lot, I found out almost all modern terminals supports 256 colors [[1]](https://github.com/termstandard/colors).
I tried changing the highlightString variable to return orange colored string and it worked. 
Sorry for not figuring it out sooner.
```go
var HighlightString = func(a ...interface{}) string {
	return "\033[38;5;202m" + fmt.Sprint(a...) + "\033[0m"
}
```
![image](https://github.com/keploy/keploy/assets/91385321/1bca4215-4e0e-451c-b621-3266b9723436)

I can create a PR for it, if you like !
